src/app/pages/product/product.component.scss

```diff
@@ -1,8 +1,9 @@
 @import '../../../styles/colors';
+@import '../../../styles/breakpoints';
 
 .product-detail {
   display: flex;
-  align-items: center;
+  align-items: flex-start;
```

Comment:
Why flex-start?

Reply:
To keep it at the first line, in case that the description length ins too long.